src/bin/worker.js

```diff
@@ -122,6 +119,16 @@ program.parse(process.argv);
     config = _.defaultsDeep(targetConfig, config);
   }
 
+  process.on('unhandledRejection', async (reason, p) => {
```

Comment:
Would it make sense to use `process.once` here, so if there's a second unhandled error it just kills the process?